nixos/modules/services/web-apps/nextcloud.nix

```
@@ -218,6 +218,12 @@ in {
       '';
     };
 
+    enableCompressedAssets = mkOption {
+      type = types.bool;
+      default = true;
+      description = lib.mdDoc "Whether to use gzip and brotli pre-compressed artifacts for lower CPU usage and transfer size but bigger storage size.";
```

```suggestion
      description = lib.mdDoc "Whether to use gzip and brotli pre-compressed assets for lower CPU usage in production and less transfer size.";
```
the last is only true after moving the compression into passthru